### ISSUE TITLE
Separate charts into their own stylesheet?

### ISSUE THREAD

**Issue by dukke** [JFXtras/jfxtras-styles#109]

**Comment by Kilobyte1000** [comment]
Does this mean you're adding chart styles soon?

**Comment by dukke** [comment]
No, this is just about potentially putting them in a separate stylesheet. 
Do you feel a need for new chart styles (yeah JMetro still doesn't have them..)?

**Comment by Kilobyte1000** [comment]
Actually, I have not used charts much. About pie chart (which I have used) replacing the gradient fills with solid colors might just be enough
